addons/website/tests/test_snippets.py

```diff
@@ -29,8 +29,17 @@ def test_01_empty_parents_autoremove(self):
     def test_02_default_shape_gets_palette_colors(self):
         self.start_tour('/@/', 'default_shape_gets_palette_colors', login='admin')
 
-    @unittest.skip
     def test_03_snippets_all_drag_and_drop(self):
+        self.env.ref('website.default_website').write({
+            'social_facebook': "https://www.facebook.com/Odoo",
+            'social_twitter': 'https://twitter.com/Odoo',
+            'social_linkedin': 'https://www.linkedin.com/company/odoo',
+            'social_youtube': 'https://www.youtube.com/user/OpenERPonline',
+            'social_github': 'https://github.com/odoo',
+            'social_instagram': 'https://www.instagram.com/explore/tags/odoo/',
+            'social_tiktok': 'https://www.tiktok.com/@odoo',
+            'social_discord': 'https://discord.com/servers/discord-town-hall-169256939211980800',
+        })
```

Comment:
Why do you need this change ?

Reply:
@FrancoisGe The reason to add that is this tour drops s_social_media snippets and if the values are not provided to social_medias the value of props for component 'BuilderTextInputBase': 'value' will be boolean not a string or null.